scripts/install-internal.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+const flagFile = path.join(process.cwd(), '.peer-deps-installed');
+
+if (fs.existsSync(flagFile)) {
+    console.log('Peer dependencies already installed, skipping...');
+    fs.rmSync(flagFile);
+    process.exit(0);
+}
```

Comment:
Why this flagFile is needed
The install-internal.js script runs after `yarn` or `npm` install. Since this script also installs dependencies, it would trigger itself again, causing an infinite loop. To prevent this, we use a flagFile.
I’m fine with implementing it differently.
Another option could be to call install-internal.js in the build scripts after yarn. I tried to avoid aditional steps, but this solution is not optimal either.